src/Console/Command/WorkerCommand.php

```diff
@@ -0,0 +1,269 @@
+    private ReadonlyCacheManager $readonlyCacheManager;
+
+    /** @var array<int, FixerFileProcessedEvent> */
+    private array $events;
+
+    public function __construct(ToolInfoInterface $toolInfo)
+    {
+        parent::__construct();
+
+        $this->setHidden(true);
+        $this->toolInfo = $toolInfo;
+        $this->errorsManager = new ErrorsManager();
+        $this->eventDispatcher = new EventDispatcher();
+    }
+
+    protected function configure(): void
+    {
+        $this->setDefinition(
+            [
+                new InputOption(
+                    'port',
+                    null,
+                    InputOption::VALUE_REQUIRED,
+                    'Specifies parallelisation server\'s port.'
+                ),
+                new InputOption(
+                    'identifier',
+                    null,
+                    InputOption::VALUE_REQUIRED,
+                    'Specifies parallelisation process\' identifier.'
+                ),
+                new InputOption(
+                    'allow-risky',
+                    '',
+                    InputOption::VALUE_REQUIRED,
+                    'Are risky fixers allowed (can be `yes` or `no`).'
+                ),
+                new InputOption('config', '', InputOption::VALUE_REQUIRED, 'The path to a config file.'),
+                new InputOption(
+                    'dry-run',
+                    '',
+                    InputOption::VALUE_NONE,
+                    'Only shows which files would have been modified.'
+                ),
+                new InputOption(
+                    'rules',
+                    '',
+                    InputOption::VALUE_REQUIRED,
+                    'List of rules that should be run against configured paths.'
+                ),
+                new InputOption(
+                    'using-cache',
+                    '',
+                    InputOption::VALUE_REQUIRED,
+                    'Should cache be used (can be `yes` or `no`).'
+                ),
+                new InputOption('cache-file', '', InputOption::VALUE_REQUIRED, 'The path to the cache file.'),
+                new InputOption('diff', '', InputOption::VALUE_NONE, 'Prints diff for each file.'),
+            ]
+        );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $errorOutput = $output instanceof ConsoleOutputInterface ? $output->getErrorOutput() : $output;
+        $identifier = $input->getOption('identifier');
+        $port = $input->getOption('port');
+
+        if (null === $identifier || !is_numeric($port)) {
+            throw new ParallelisationException('Missing parallelisation options');
+        }
+
+        try {
+            $runner = $this->createRunner($input);
+        } catch (\Throwable $e) {
+            throw new ParallelisationException('Unable to create runner: '.$e->getMessage(), 0, $e);
```

Comment:
We _do_ this before in `FixCommand` too, but this one is not about configuration on project level, but about the way how `WorkerCommand` is run. Yes, it's related to how main command was executed, but it's more about how process factory work and how worker command was executed in child process... For now it only throws an exception if `--config` and `--rules` were passed at the same time, but maybe it will have more conditions in the future? Can't tell, but we need to catch this and report it in main process.